### DISCUSSION TITLE
Will order LastUpdateDate update when order is refunded?

### DISCUSSION THREAD

**Discussion by DanielLanger** [amzn/selling-partner-api-models#3299] [General]
Will an orders LastUpdateDate (https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#order) update when an order is refunded? I don't see a refunded/ returned status itself, but I am curious if that date will update still. I am looking for a way to know when an order has been refunded, so I can THEN call listFinancialEventsByOrderId (https://developer-docs.amazon.com/sp-api/docs/finances-api-reference#listfinancialeventsbyorderid) on that order to get the details.

**Comment by wszwc** [comment]
Hello, I am also confirming this problem. I want to know whether the information in GetOrders will change after the financial information is updated. I need to get MD5 from GetOrders to determine whether the order is updated. If the financial information changes, the information in GetOrders will not change. change, it would be a disaster for me
